StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentMethodFormViewController.swift

```diff
@@ -5,40 +5,142 @@
 //  Created by Yuki Tokuhiro on 5/16/24.
 //
 
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
 @_spi(STP) import StripeUICore
 import UIKit
 
+protocol PaymentMethodFormViewControllerDelegate: AnyObject {
+    func didUpdate(_ viewController: PaymentMethodFormViewController)
+    func updateErrorLabel(for error: Error?)
+}
+
 class PaymentMethodFormViewController: UIViewController {
+    /// This caches forms for payment methods so that customers don't have to re-enter details
+    /// This class expects the formCache to be invalidated (cleared) when we load PaymentSheet; we assume the form generated for a given PM type _does not change_ at any point after load.
+    static var formCache: [PaymentSheet.PaymentMethodType: PaymentMethodElement] = [:]
```

Comment:
Struggling to see why this needs to be static

Reply:
We create instances of this class ~every time we go to the PM form, and the cache needs to live longer than that.